plugins/statistics/index.js

```diff
@@ -19,33 +26,13 @@ module.exports = {
         tags: ["api", "statistics", "non-critical"]
       },
       handler: async (request, h) => {
-        const db = request.server.app.db;
-
-        const options = {
-          reduce: "true"
-        };
-
-        if (request.params.since) {
-          options.startkey = request.params.since;
-        }
-
-        const returnValue = await new Promise((resolve, reject) => {
-          db.view("items", "numberOfItems", options, (err, data) => {
-            if (err) {
-              return reject(Boom.internal(err));
-            } else {
-              if (data.rows.length === 0) {
-                return resolve({
-                  value: 0
-                });
-              }
-              return resolve({
-                value: data.rows[0].value
-              });
-            }
-          });
+        return request.server.methods.db.statistics.getNumberOfItems({
```

Comment:
Don't you have to await the result of the server method here? => `return await request.server. ...`

Reply:
No, although it doesn't need to be an async function (I'll correct that). Hapi correctly handles a Promise as a return value of a handler.

Reply:
I see 👍